### PR TITLE
fix(dropdown): fix default select

### PR DESCRIPTION
**Pull request by EmelieLitwin**
**Describe pull-request**  
The dropdown option default select picks the wrong default option if one of the values is included in another.
Example:
```
export default function App() {
  const data = `[
    {"label": "Denmark","value":"1384"},
    {"label": "Sweden","value":"4"}]`;
  return (
    <div className="App">
      {/* 1384 is Denmark, but Sweden is selected in the dropdown*/}
      <sdds-dropdown-filter data={data} default-option="1384" />
    </div>
  );
}
```
See codebox for example: https://codesandbox.io/s/tegel-dropdown-test-wqzyqo?file=/src/App.tsx

**Solving issue**  
Solving github issue #493 and Azure task [AB#2427](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2427)

**How to test**  
I tested it by including the provided code example in storybook and made sure it worked, but didn't push that. 
Start with looking at the code and see if it's a potentially good solution.

**Screenshots**  
image

### CODE REVIEW

File: tegel/src/components/dropdown/dropdown.tsx [hunk 1]
```diff
@@ -110,6 +110,7 @@ export class Dropdown {
   setOptionFromOutside(optionValue) {
     if (optionValue) {
       this.deselectAll();
+      optionValue = optionValue.split(',');
       for (let i = 0; i < this.host.children.length; i++) {
         const el = this.host.children[i];
         if (optionValue.includes(el['value']) || el['value'] === optionValue) {
```
Comment (mJarsater) [inline tegel/src/components/dropdown/dropdown.tsx:116]:
```suggestion
        if (el['value'] === optionValue) {
```

[the remark this answers is not in the document]
Comment (theJohnnyMe) [reply tegel/src/components/dropdown/dropdown.tsx:116]: I agree. Right now, I was trying to use values 44 and 4, if I say default-option="4", for every option that contains the number 4 is pre-selected.

**Comment by theJohnnyMe** [comment]
PS: Nice that the user provided codesandbox, however, there was a version 5 with a beta tag that is old and does not bring changes done in 4.10.1 - I changed it, but still issue is there. 
It is more of heads-up.

File: components/src/components/dropdown/dropdown.tsx [hunk 2]
```diff
@@ -110,6 +110,7 @@ export class Dropdown {
   setOptionFromOutside(optionValue) {
     if (optionValue) {
       this.deselectAll();
+      optionValue = optionValue.split(',');
       for (let i = 0; i < this.host.children.length; i++) {
         const el = this.host.children[i];
         if (optionValue.includes(el['value']) || el['value'] === optionValue) {
```
Comment (mJarsater) [inline components/src/components/dropdown/dropdown.tsx:116]:
```suggestion
        if (el['value'] === optionValue) {
```

**Review by theJohnnyMe — APPROVED** [review]
LGTM

**Pull request merged**